packages/react-jsx-highcharts/src/components/UseSeries/index.js

```diff
@@ -1,16 +1,15 @@
 import { useContext, useState, useDebugValue } from 'react';
 import SeriesContext from '../SeriesContext';
-import createProvidedSeries from '../Series/createProvidedSeries';
 import useChart from '../UseChart';
+import createProvidedSeries from '../Series/createProvidedSeries';
 import useDelayOnce from '../UseDelayOnce';
 
 export default function useSeries(seriesId) {
-  const providedSeries = useContext(SeriesContext);
-
+  const contextSeries = useContext(SeriesContext);
   const chart = useChart();
 
   const createStateSeries = () => {
-    if (providedSeries) return createProvidedSeries(providedSeries);
+    if (contextSeries) return createProvidedSeries(contextSeries);
```

Comment:
This differs from `UseAxis`, [the create function isn't called](https://github.com/whawker/react-jsx-highcharts/blob/master/packages/react-jsx-highcharts/src/components/UseAxis/index.js#L12). Is this intentional?

Reply:
Nice find! It is definitely a bug.

Reply:
The useAxis is correct.